tests/Unit/Helpers/DataStructures/Tensor/Expressions/EvaluateRank4TestHelpers.hpp

```diff
@@ -12,6 +12,9 @@
 #include "DataStructures/Tensor/Tensor.hpp"
 #include "Utilities/TMPL.hpp"
 
+namespace TestHelpers {
```

Comment:
clang-tidy will want you to do `TestHelpers::TensorExpressions`. This type of namespace scoping is a C++17 feature :)

Reply:
Done!